.github/workflows/managed-java-checks.yaml

```diff
@@ -0,0 +1,38 @@
+name: Lint java code under managed
+
+on:
+  push:
+    branches:
+      - master
+      - "2.6"
```

Comment:
Don't know if double quote or single quote matters, but for consistency, maybe use single quotes?